src/analytics/Properties.tsx

```diff
@@ -1626,7 +1626,6 @@ interface EarnEventsProperties {
   [EarnEvents.earn_withdraw_add_gas_press]: EarnCommonProperties & { gasTokenId: string }
   [EarnEvents.earn_info_learn_press]: undefined
   [EarnEvents.earn_info_earn_press]: undefined
-  [EarnEvents.earn_active_pools_card_press]: undefined
```

Comment:
Probably fine removing this, but this could be a potential backwards incompatible change on analytics if there's some dashboard that relies on this as this is removed and the meaning of `earn_entrypoint_press` has changed slightly

Reply:
I could keep `earn_active_pools_card_press` and use that event for when the card is pressed and the user has pools supplied, but that seems a bit confusing since the card is no longer called "Active Pools Card" for this case. Should I check with product about the dashboards?

Reply:
yes